app/workers/publishing_api_unpublishing_worker.rb

```diff
@@ -0,0 +1,41 @@
+class PublishingApiUnpublishingWorker
+  include Sidekiq::Worker
+
+  def perform(unpublishing_id, locale = I18n.default_locale.to_s)
+    unpublishing = Unpublishing.includes(edition: [:document]).find(unpublishing_id)
+    edition = unpublishing.edition
+    document = edition.document
```

Comment:
```ruby
content_id = Document.where(id: edition.document_id).pluck(:content_id).first
```

We don't need document for anything else, and it's quite expensive to instantiate. Can then also reduce the `includes` above.